core-kotlin-modules/core-kotlin-6/README.md

```diff
@@ -2,3 +2,4 @@
 
 - [Deprecation in Kotlin](https://www.baeldung.com/kotlin/deprecation)
 - [Measuring Elapsed Time in Kotlin](https://www.baeldung.com/kotlin/measure-elapsed-time)
+- [Kotlin Bit Operations](https://www.baeldung.com/kotlin/kotlin-bit-operations)
```

Comment:
You don't have to add this yet, since the link is still TBD, this gets added after the article is published.

Reply:
Noted

Reply:
For now it's safe to just delete this line 😄 

```suggestion
```